src/index.scss

```
@@ -1,6 +1,7 @@
+@import "../src/styles/normalize";
+
 * {
-  margin: 0px;
-  padding: 0px;
+  box-sizing: border-box;
```

please also add these rules, this will make sure that all elements don't use values provided by the browser

```suggestion
  box-sizing: border-box;
  padding: 0;
  margin: 0;
```

After this is updated, you can remove styles like:

```scss
.movies-list {
  margin-bottom: 0;
```